elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -76,15 +76,15 @@ class PfeNavigation extends PFElement {
 
   static get properties() {
     return {
-      _lang: {
```

Comment:
@TylerMartin5 We weren't sure what the reason was for the underscore, I don't think an HTML attribute can start with an underscore, so we thought it best to remove it?